tutor-multi-chart/values.yaml

```diff
@@ -20,46 +19,95 @@
-   # Eventually this should be fixed upstream in the k8s scheduler: https://github.com/kubernetes/kubernetes/issues/103305
-   affinity:
-      podAffinity:
-         requiredDuringSchedulingIgnoredDuringExecution:
-            - labelSelector:
-               matchLabels:
-                  app.kubernetes.io/name: '{{ template "traefik.name" . }}'
-                  app.kubernetes.io/instance: '{{ .Release.Name }}-{{ .Release.Namespace }}'
-              topologyKey: kubernetes.io/hostname
+  # See https://github.com/traefik/traefik-helm-chart/blob/master/traefik/values.yaml
+  # for all available options
+  certResolvers:
+     letsencrypt:
+        # email: email@example.com
+        httpChallenge:
+           entryPoint: "web"
+        storage: /data/acme.json # match the path to persistence (below)
+  persistence:
+     # Enable persistence at /data to store letsencrypt status
+     enabled: true
+     path: /data
+     # DigitalOcean minimum size for a persistent volume claim is currently 1GB
+     size: 1Gi
+  ports:
+     web:
+        redirectTo: websecure
+     websecure:
+        tls:
+           enabled: true
+           certResolver: "letsencrypt"
+  deployment:
+     replicas: 1 # Only 1 replica is supported if using Lets Encrypt
+     # The "volume-permissions" init container is required to avoid permission issues.
+     # Related issue: https://github.com/containous/traefik/issues/6972
+     initContainers:
+        - name: volume-permissions
+          image: busybox:1.31.1
+          command: ["sh", "-c", "chmod -Rv 600 /data/*"]
+          volumeMounts:
+            - name: data
+              mountPath: /data
+  # When deploying a new version of this pod, it must be deployed to the same node, because of the ReadWriteOnce
+  # volume, which can only be mounted on one node at a time.
+  # Eventually this should be fixed upstream in the k8s scheduler: https://github.com/kubernetes/kubernetes/issues/103305
+  affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - topologyKey: kubernetes.io/hostname
+          labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: '{{ template "traefik.name" . }}'
+              app.kubernetes.io/instance: '{{ .Release.Name }}-{{ .Release.Namespace }}'
+
+
+# Multi-tenant ElasticSearch
+elasticsearch:
+  enabled: false
+
+  # Operators will need to add/update the following setting in each
+  # of their instances by running the commands:
+  # ```
+  # tutor config save --set K8S_HARMONY_ENABLE_SHARED_ELASTICSEARCH=true --set RUN_ELASTICSEARCH=false
+  # tutor harmony create-elasticsearch-user
+  # ```
+  # RUN_ELASTICSEARCH: false
+  # ELASTICSEARCH_PREFIX_INDEX: "username-"
+  # MULTI_K8S_USE_SHARED_ELASTICSEARCH: true
+  # ELASTICSEARCH_AUTH: "username:actual_password"
+
+  # The chart will handle certificate creation so that the CA can be used
+  # in other pods/namespaces.
+  createCert: false
```

Comment:
The comment says "The chart will handle certificate creation" but `createCert: false` sounds like we're telling the chart _not_ to handle it?

Reply:
Ah thanks for pointing that out. The wording was confusing. I meant that the `k8s-harmony` chart would create the certs.